src/H3MP/Plugin.cs

```diff
@@ -49,6 +54,10 @@ public class Plugin : BaseUnityPlugin
 		private readonly ManualLogSource _serverLog;
 		private readonly ManualLogSource _discordLog;
 
+		private Text[] _panelTextArray;
+		private Text _panelTitle;
+		private Text _panelBody;
```

Comment:
Might be a better idea to make a class for this, considering that the names all start with `panel`.